modules/network-observability-kafka-option.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+
+// * installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-kafka-option_{context}"]
+
+= Installing Kafka (optional)
+The Kafka Operator is supported for large scale environments. You can install the Kafka Operator as Red Hat AMQ Streams from the Operator Hub, just as the Loki Operator and Network Observability Operator were installed. 
```

Comment:
Comparing Kafka with Loki and NOO is misleading for me since these doesn't have the same purpose at all.

We should explain the value of using Kafka here, the minimum compatible version and provide & explain instance / topic / user yamls

Here is the upstream doc: https://github.com/netobserv/documents/blob/main/kafka.md

@jotak @OlivierCazade @mariomac thoughts ?

Reply:
The benefits of using Kafka are mentioned there already: https://github.com/openshift/openshift-docs/pull/53360/files#diff-f1449138f2d1175df0811b265917416523d8c44352f06cdb14b13a11db3fca11R22

I think we could point to AMQ Streams doc, but we shouldn't start replicating it.

Reply:
@skrthomas can we link to https://access.redhat.com/documentation/en-us/red_hat_amq_streams/2.2 ?

Reply:
@jotak @jpinsonneau i added Joel’s topic about configuring the flow collector resource if you’re using Kafk, see https://53263--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/configuring-operator.html#configuring-the-flow-collector-resource-with-kafka. It has the link to the AMQ documentation and has user YAML with callouts-thanks Joel! Not sure if this is what you had in mind, @jpinsonneau? I realize I mentioned Joel’s topic in my Slack but didn’t link there.

Reply:
Added the link to AMQ here.